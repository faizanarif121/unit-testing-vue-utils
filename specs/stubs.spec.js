import { shallowMount } from '@vue/test-utils'
import ListComponent from '@/list'

const ListItemComponent = {
  template: `<li>  {{movie}} </li> `,
  props: ['movie'],
}

describe('Play with Child Components', () => {
  test('shallow mount component', () => {
    const wrapper = shallowMount(ListComponent, {
      stubs: {
        ListItem: ListItemComponent,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
