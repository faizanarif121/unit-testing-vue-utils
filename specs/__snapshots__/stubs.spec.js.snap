// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Play with Child Components shallow mount component 1`] = `
<div>
  <ul>
    <li> Iron Man </li>
    <li> Avengers </li>
    <li> Infitinity War </li>
  </ul>
</div>
`;
